code/search/SearchUpdater.php

```diff
@@ -90,14 +90,14 @@ public static function handle_manipulation($manipulation)
         $writes = array();
 
         foreach ($manipulation as $table => $details) {
-            if (!isset($details['id']) || !isset($details['fields'])) {
+            if (!isset($details['id'])) {
```

Comment:
if `fields` is empty, we still need to run this loop to trigger the `update to more specific classname` line below.